core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListResourceUtils.java

```diff
@@ -0,0 +1,134 @@
+     * container with given properties.
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param parent           Container for {@code listItem}'s.
+     * @param properties       {@code listItem} properties
+     * @throws PersistenceException if {@code listItem} cannot be created
+     */
+    static void createListItem(ResourceResolver resourceResolver, Resource parent, Map<String, Object> properties) throws PersistenceException {
+        Map<String, Object> withoutSystemProperties = getWithoutSystemProperties(properties);
+        withoutSystemProperties.put(JcrResourceConstants.SLING_RESOURCE_TYPE_PROPERTY, ListConstants.LIST_ITEM_RESOURCE_TYPE);
+        resourceResolver.create(parent, ResourceUtil.createUniqueChildName(parent, CoreConstants.PN_LIST_ITEM), withoutSystemProperties);
+    }
+
+    /**
+     * Converts collection of {@link SimpleListItem} to list of {@link ValueMapResource}
+     * @param values collection of {@link SimpleListItem} that will be converted to {@link ValueMapResource}
+     * @return list of {@link ValueMapResource}
+     */
+    static List<Resource> mapToValueMapResources(Collection<SimpleListItem> values) {
+        return CollectionUtils.emptyIfNull(values).stream()
+            .map(entry -> ListResourceUtils.createValueMapResource(entry.getTitle(), entry.getValue()))
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Converts key-value map to list of {@link ValueMapResource}
+     * @param values key-value map that will be converted to {@link ValueMapResource}
+     * @return list of {@link ValueMapResource}
+     */
+    static List<Resource> mapToValueMapResources(Map<String, Object> values) {
+        return MapUtils.emptyIfNull(values).entrySet().stream()
+            .map(entry -> ListResourceUtils.createValueMapResource(entry.getKey(), entry.getValue()))
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Creates {@link ValueMapResource} representation of {@code listItem} entry using title as a {@code jcr:title}
+     * and {@code value} as a value
+     * @param title {@code jcr:title} of {@code listItem}
+     * @param value {@code value} of {@code listItem}
+     * @return {@link ValueMapResource} representation of {@code listItem}
+     */
+    static Resource createValueMapResource(String title, Object value) {
+        Map<String, Object> properties = new HashMap<>();
+        properties.put(JcrConstants.JCR_TITLE, title);
+        properties.put(CoreConstants.PN_VALUE, value);
+        return new ValueMapResource(null, "", JcrConstants.NT_UNSTRUCTURED, new ValueMapDecorator(properties));
+    }
+
+    /**
+     * Returns a map without ignored properties
+     * @param properties initial map of properties
+     * @return filtered map of properties
+     */
+    private static Map<String, Object> getWithoutSystemProperties(Map<String, Object> properties) {
```

Comment:
Suggestion to rename method from `getWithoutSystemProperties` to `getMapWithoutSystemProperties`